packages/design-system-react/package.json

```diff
@@ -40,7 +40,7 @@
     "changelog:validate": "../../scripts/validate-changelog.sh @metamask/design-system-react",
     "clean:icons": "rimraf src/components/Icon/icons",
     "create-component": "ts-node scripts/create-component",
-    "generate:icons": "yarn clean:icons && svgr --config-file src/components/Icon/.svgrrc.js -d src/components/Icon/icons src/components/Icon/assets/*.svg && yarn generate:icons:index",
+    "generate:icons": "yarn clean:icons && svgr --config-file src/components/Icon/.svgrrc.js -d src/components/Icon/icons src/components/Icon/assets/*.svg && yarn generate:icons:index && yarn workspace @metamask/metamask-design-system prettier --write \"packages/design-system-react/src/components/Icon/icons/**/*.{ts,tsx}\"",
```

Comment:
Adding prettier formatting step after SVGR icon generation since we've [disabled SVGR's built-in prettier](https://github.com/MetaMask/metamask-design-system/pull/546/files#r2051959641) (due to ESM/CommonJS conflicts with prettier-plugin-tailwindcss). This ensures our icons are formatted consistently with our project's prettier config while avoiding the module system conflicts. See discussion in [svgr#893](https://github.com/gregberge/svgr/issues/893) and [PR discussion](https://github.com/MetaMask/metamask-design-system/pull/546/files#r2051959641).